package.json

```diff
@@ -14,8 +14,7 @@
   "dependencies": {
     "css-vendor": "^0.3.1",
     "debug": "^2.1.3",
-    "lodash.throttle": "^3.0.3",
-    "object.assign": "^4.0.1"
```

Comment:
Another solution that I have so far opted against is to let the user supply their own polyfill but I think its less friendly for the user. Either they have to read the README which states this or we use NPM Peer Dependencies which still suffers from forcing us to choose which library to use.